EIPS/eip-3041.md

```diff
@@ -0,0 +1,95 @@
+---
+eip: 3041
+title: Add basefee in eth_getBlockByHash response
+author: Abdelhamid Bakhta (@abdelhamidbakhta)
+discussions-to: https://ethereum-magicians.org/t/eip-3041-add-basefee-in-eth-getblockbyhash-response/4825
+status: Draft
+type: Standards Track
+category : Interface
+created: 2020-10-13
+requires: 1559
+---
+
+## Simple Summary
+Add basefee field in `eth_getBlockByHash` RPC endpoint response.
+
+## Abstract
+[EIP-1559](/EIPS/eip-1559) introduces a base fee per gas in protocol.
+This value is maintained under consensus as a new field in the block header structure.
+This change should be reflected in RPC methods that return block header information.
+
+## Motivation
+Users may need value of the base fee at a given block. Base fee value is important to make gas price predictions more accurate.
+
+## Specification
+
+### eth_getBlockByHash
+
+#### Description
+
+Returns information about a block specified by hash.
+
+#### Parameters
+
+Parameters remain unchanged.
+
+#### Returns
+Add a new JSON field in the `result` object for block headers containing a base fee (post [EIP-1559](/EIPS/eip-1559) fork block).
```

Comment:
```suggestion
Add a new JSON field in the `result` object for block headers containing a base fee (post [EIP-1559](./eip-1559.md) fork block).
```
Please use this format for links to other EIPs, as it works in all known environments while other forms of the path break in some environments.